pkg/agent/templates/templates.go

```diff
@@ -68,6 +68,7 @@ const ContainerdConfigTemplate = `
   {{ if $v.TLS.CAFile }}ca_file = "{{ $v.TLS.CAFile }}"{{end}}
   {{ if $v.TLS.CertFile }}cert_file = "{{ $v.TLS.CertFile }}"{{end}}
   {{ if $v.TLS.KeyFile }}key_file = "{{ $v.TLS.KeyFile }}"{{end}}
+  {{ if $v.TLS.InsecureSkipVerify }}insecure_skip_verify = "{{ $v.TLS.InsecureSkipVerify }}"{{end}}
```

Comment:
for toml, I believe this should be `true` or `false`, not a quoted string. Also, since we're checking a boolean, using a fixed value if it's true, and omitting it if it's false, should be fine.
```suggestion
  {{ if $v.TLS.InsecureSkipVerify }}insecure_skip_verify = true{{end}}
```

Reply:
Sorry for this stupidity.. was typing on my phone so I just copy-pasted stuff :confounded: